<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ke6221</class>
 <widget class="QDialog" name="ke6221">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_RangeType">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Range Type</string>
   </property>
  </widget>
  <widget class="QComboBox" name="RangeType">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Auto</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Manual</string>
    </property>
   </item>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>451</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>IF MANUAL</string>
   </property>
   <widget class="QComboBox" name="SetRange">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>2 nA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20 nA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>200 nA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2 μA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20 μA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>200 μA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2 mA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20 mA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100 mA</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_SetRange_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Range</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_OutPut">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Schoolbook L</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Output</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="Output">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>8</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Name">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 16pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>KE6221</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_amplitude">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Amplitude</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_SetFilter_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Fiter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SetFilter">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>130</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top: 3px transparent;
            border-bottom: 3px transparent;
            border-right: 10px transparent;
            border-left: 10px transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/on-off.jpg</normaloff>:/on-off.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>81</width>
     <height>31</height>
    </size>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="SetAmplitude">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="maximum">
    <double>100000.000000000000000</double>
   </property>
  </widget>
  <widget class="QComboBox" name="RangeType_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>90</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>mA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>μA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>nA</string>
    </property>
   </item>
  </widget>
  <widget class="QDoubleSpinBox" name="SetCompliance">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>0.100000000000000</double>
   </property>
   <property name="maximum">
    <double>105.000000000000000</double>
   </property>
   <property name="value">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_Compliance">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Compliance</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>250</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
